src/Domain/FeedbackGetDomain.php

```diff
@@ -29,22 +30,19 @@ public function __invoke(array $input)
                 $this->initRedis();
             }
 
-            $feedbackItems = $this->getFeedbackItems();
+            $rawFeedbacks = $this->getRawFeedbacks();
 
-            if (count($feedbackItems) > 0 && !$debug) {
-                $this->setRedisValue($this->getValueForRedis(reset($feedbackItems)));
+            if (count($rawFeedbacks) > 0 && !$debug) {
```

Comment:
You don't actually need a `count` comparison here. Using `if ($rawFeedbacks && ...)` would be enough.